lib/components/home/artwork_rails/artwork_rail_header.js

```diff
@@ -56,23 +56,31 @@ class ArtworkRailHeader extends React.Component {
     }
   }
 
+  hasAdditionalContent() {
+    const additionalContentRails = [
+      'followed_artists',
+      'saved_works',
+      'live_auctions',
+      'current_fairs',
+      'genes',
+      'generic_gene',
+    ]
```

Comment:
Yes, and I was thinking about a way to share this information between the two, but I didn't want to add too much complexity at first.